app/src/main/res/layout/activity_main.xml

```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v4.widget.NestedScrollView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
imo with linebreak is looking better

Reply:
@sleepylee hmm I think we should define an IDE config for internal team using to avoid this conflict 🤔